Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -130,6 +130,33 @@ void LoadBankFromDiskTask::prepareEventId(::NeXus::File &file, int64_t &start_ev
   int64_t dim0 = recalculateDataSize(id_info.dims[0]);
   stop_event = dim0;
 
+  // Handle the time filtering by changing the start/end offsets.
```

Comment:
I'm not sure the change to this file belongs back here. @DannyHindson fixed a bug with filtering by time in `LoadEventNexus` if there isn't a cast-iron guarantee that events in the file are sorted by pulse time. This was originally done on `release-next` and then merged to `main` so maybe some there was some mixup there?

Reply:
Can @DannyHindson look at the file that results from the merge and see if the code belongs? I think it got incorrectly removed the last time I merged `ornl-next` into `main` so this would put it back.